src/AccessibilityInsights.Rules/EvaluationCode.cs

```diff
@@ -8,12 +8,42 @@ namespace Axe.Windows.Rules
     /// </summary>
     public enum EvaluationCode
     {
-        NotApplicable, // given element did not meet the conditions for the rule
+        /// <summary>
+        /// The given element did not meet the conditions for the rule and was not evaluated.
+        /// </summary>
+        NotApplicable,
+
+        /// <summary>
+        /// The given element met the success criteria  of the rule evaluation.
+        /// </summary>
         Pass,
-        Error, // fail
-        Open, // not enough information to make a determination
-        Note, // informational
+
+        /// <summary>
+        /// The given element did not meet the success criteria  of the rule evaluation.
+        /// </summary>
+        Error,
+
+        /// <summary>
+        /// The rule is unable to make a determination, and the element should be investigated.
+        /// </summary>
+        Open,
+
+        /// <summary>
+        /// The rule's description is informational and does not indicate success or failure.
+        /// </summary>
+        Note,
+
+        /// <summary>
+        /// The given element did not meet the success criteria  of the rule evaluation,
```

Comment:
There is an extra whitespace character between 'criteria' and 'of the rule evaluation'